content/LGinC/drone-portainer/index.md

```diff
@@ -0,0 +1,64 @@
+---
+date: 2020-07-16T00:00:00+00:00
+title: "Portainer Stack Deploy"
+author: "LGinC"
+tags: [ deploy ]
+repo: "turinge/drone-portainer"
+logo: "portainer.svg"
+image: "turinge/drone-portainer"
+---
+
+The drone plugin can be used to deploy a Docker image to a Drone environment.The below pipeline configuration demonstrates simple usage:
```

Comment:
Docker image to a Drone environment?

Reply:
content of Dockerfile

```
FROM alpine

ADD "entrypoint.sh" "/entrypoint.sh"
RUN sed -i "s@dl-cdn.alpinelinux.org@mirrors.aliyun.com@g" /etc/apk/repositories && \
 apk -Uuv --no-cache add curl jq bash ca-certificates && chmod +x entrypoint.sh
ENTRYPOINT ["/entrypoint.sh"]
```

Reply:
Deploy a docker image to drone environment doesn't make sense, I don't get what you want to say with that.

Reply:
deploy docker-compose to portainer site

```
docker_compose: |
  version: "2"
  myapi:
    image: xxx/xx
```

this operation will generate a stack in portainer

Reply:
[https://app.swaggerhub.com/apis/deviantony/Portainer/1.23.2/#/stacks/StackCreate](https://app.swaggerhub.com/apis/deviantony/Portainer/1.23.2/#/stacks/StackCreate)

Reply:
Than it should be still something like thsi:

```
This plugin can be used to deploy a Docker image to Portainer. The below pipeline configuration demonstrates simple usage:
```

Reply:
done